stock_move_manual_lot/models/stock_picking.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    use_manual_lot_selection = fields.Boolean(
+        related="picking_type_id.use_manual_lot_selection",
+    )
+
+    def button_validate(self):
```

Comment:
Seem to have missed this earlier, but I would appreciate a docstring clarifying what this does.

Reply:
[done](https://github.com/OCA/stock-logistics-workflow/pull/833/commits/94059211c9a410ad3fd7a3ac9a4e6fd89d71da53#diff-e6d83dd27f05ea77e0d86d96158521c823c7e9c6f662f885effedd1e8a9bc7c9R14)